### PR TITLE
Split out DLP from MIP

### PR DESCRIPTION
**Pull request by tiffanyyumsft**

### CODE REVIEW

File: microsoft-365/compliance/compliance-easy-trials.md [hunk 1]
```diff
@@ -70,6 +70,14 @@ Wondering what you can experience in your free trial? The compliance solutions t
 
   Compliance Manager can help you throughout your compliance journey, from taking inventory of your data protection risks to managing the complexities of implementing controls, staying current with regulations and certifications, and reporting to auditors. [Learn more about Compliance Manager](compliance-manager.md)
 
+- **Data Loss Prevention (DLP)** 
+  Data Loss Prevention helps you control sharing and use of sensitive info on devices, apps, and services. With trials, you can extend coverage of DLP polices to Teams and devices, protect even more sensitive content using trainable classifiers and exact data match (EDM), and get started quickly with our recommended DLP policies.  [Learn more about Data Loss Prevention](dlp-learn-about-dlp.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-easy-trials.md]:
```suggestion
- **Data Loss Prevention** 
  Data Loss Prevention (DLP) helps you control sharing and use of sensitive info on devices, apps, and services. With trials, you can extend coverage of DLP policies to Teams and devices, protect even more sensitive content using trainable classifiers and exact data match (EDM), and get started quickly with our recommended DLP policies.  [Learn more about Data Loss Prevention](dlp-learn-about-dlp.md)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tiffanyyumsft, kindly commit the suggested changes. Many thanks.

**Comment by tiffanyyumsft** [comment]
> Copy review - @tiffanyyumsft, kindly commit the suggested changes. Many thanks.

@JohanFreelancer9 Done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrfox** [comment]
adding @v-tophillips

**Review by v-tophillips — APPROVED** [review]

**Review by v-tophillips — COMMENTED** [review]
@tiffanyyumsft After looking at your changes, several places would replace the Purview branding and undo the rebranding changes. I propose canceling this PR and I will create another PR that includes your changes without the branding so that we don't undo work that is already done. Will that work for you?

**Comment by tiffanyyumsft** [comment]
@v-tophillips Yes, Todd that works for me. Appreciate your help on this!

**Pull request closed without merging**